src/docfx/config/ops/OpsConfigAdapter.cs

```diff
@@ -202,6 +186,30 @@ private async Task<string> GetMetadataSchema(Uri url)
             };
         }
 
+        private async Task<string> FetchRetry(string url, IReadOnlyDictionary<string, string> headers = null, bool nullOn404 = false)
+        {
+            try
+            {
+                using var response = await HttpPolicyExtensions
+                   .HandleTransientHttpError()
+                   .Or<OperationCanceledException>()
```

Comment:
the default http client time is 100 seconds, with 3 times retry, the total waiting time maybe 400 seconds, maybe we need consider to reduce the timeout waiting time. 
from the AI data, most of the API is finished within 1 second, so 5 seconds is enough?
![image](https://user-images.githubusercontent.com/19990166/76181306-9607ca80-61fb-11ea-8c4c-bb57f05ddaf2.png)

Reply:
This is used for both server and local, so I'll keep the default timeout.

I prefer we improve validation service stability to fix the root problem. (If the validation service is stable enough, there's probably no need to skip build in case of validation error)

Reply:
You are right, we have completed our new allow list service work, but it takes time to switch to our new service. Let's just add a simple retry logic here to make it works. Transient error can not be avoided anyway.

Reply:
I don't change the default timeout for time being, the network connectivity timeout should not  happen very frequent, and it occurs around 8 times in the past week. So I think it is still acceptable to wait Maximum 400 seconds.